internal/infrastructure/storage/db/pg/migration/20230504102232_init.up.sql

```diff
@@ -0,0 +1,63 @@
+CREATE TABLE market (
+    name VARCHAR(50) NOT NULL PRIMARY KEY,
+    base_asset VARCHAR(255) NOT NULL,
+    quote_asset VARCHAR(255) NOT NULL,
+    base_asset_precision INTEGER NOT NULL DEFAULT 8,
+    quote_asset_precision INTEGER NOT NULL DEFAULT 8,
+    tradeable BOOLEAN NOT NULL DEFAULT FALSE,
+    strategy_type INTEGER NOT NULL DEFAULT 0,
+    base_price INTEGER NOT NULL DEFAULT 0,
+    quote_price INTEGER NOT NULL DEFAULT 0,
```

Comment:
Defaults are not needed as the domain doesn't expect these fields to be nil:

```suggestion
    strategy_type INTEGER NOT,
    base_price INTEGER NOT NULL,
    quote_price INTEGER NOT NULL,
```

Reply:
note that in badger impl we decided to split market and price files as prices might be constantly updated by a feeder and the main reason was preventing conflicts in case the operator and the feeder updated the same market at nearly the same time.

Shall we have a dedicated table for prices in this pg impl? I'm not really concerned about conflicts here, but the fact that prices can be subject to multiple changes make me thinking that would be better to isolate them.

Reply:
no need to have separate table as it would be 1:1 relation, as for conflicts pg has "row lock" mechanism for updates.